cmd/stbtrace

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python2
```

Comment:
Is there a convention that source files have a language suffix? I think that would be desirable so that style checks can be automatically applied to files using a glob pattern. We can then copy/install/rename the appropriate files during a build step if needed.

On a similar note, can you run `flake8` or similar python style checker on the python files and address resulting issues? We can then work on automating that as part of PR submissions using GitHub actions going forward.

Reply:
Ran pep8 and added the .py suffix.